.github/workflows/release-new-version.yml

```diff
@@ -74,7 +74,17 @@ jobs:
             connector=$(basename ${connector_dir})
             newman dir-import ${POSTMAN_DIR}/${connector} -o ${POSTMAN_JSON_DIR}/${connector}.postman_collection.json
           done
-          (git diff --quiet && git diff --staged --quiet) || (git commit -am 'test(postman): update postman collection files' && echo "Committed changes") || (echo "Unable to commit the following changes:" && git diff)
+
+          if git diff --quiet && git diff --staged --quiet; then
+              echo "No changes detected."
+          elif git add postman; then
+              echo "Changes have been staged."
+          elif git commit -am 'test(postman): update postman collection files'; then
+              echo "Committed changes."
```

Comment:
Won't this only stage the files and not commit them if there are any unstaged files? Moreover, this is quite confusing and seems unnecessarily complicated.

Reply:
Oh! Yes! That got a bit confusing which resulted in creation of mess. Will push a fix for that in a few minutes.